.eslintrc.json

```diff
@@ -0,0 +1,41 @@
+{
+    "env": {
+        "browser": true,
+        "commonjs": true,
+        "node": true,
+        "es6": true,
+        "jest": true
+    },
+    "extends": [
+        "eslint:recommended",
+        "plugin:react/recommended"
+    ],
+    "parserOptions": {
+        "ecmaFeatures": {
+            "jsx": true
+        },
+        "ecmaVersion": 2021,
+        "sourceType": "module"
+    },
+    "plugins": [
+        "react"
+    ],
+    "rules": {
+        "indent": [
+            "error",
+            4
```

Comment:
Shouldn't it be 2?
HR style guide asks us to use 2 spaces for indentation.

Reply:
Thanks for catching that. I will make the update and push in a minute

Reply:
The indentation rule has been updated to 2 spaces on this branch @RehReis

Reply:
Thanks Aaron. This seems good to me 🙂